deploy-board/deploy_board/webapp/cluster_urls.py

```diff
@@ -20,6 +20,7 @@
 urlpatterns = [
     url(r'^clouds/create_base_image/$', cluster_view.create_base_image),
     url(r'^clouds/baseimages/$', cluster_view.get_base_images),
+    url(r'^clouds/baseimages/events/(?P<image_id>[a-zA-Z0-9\-_\.]+)/$', cluster_view.get_base_image_events),
```

Comment:
Can we move this to a different view? It has nothing to do with `cluster_view`.

Reply:
A new view may be needed. Same applies to `demote_image` and `promote_image`

Reply:
yea, make sense. I am also not sure how those image related stuff was added under cluster view. this can be done in another pr